### DISCUSSION TITLE
Scripts Saved to Individual File

### DISCUSSION THREAD

**Discussion by xG00BERx** [Reetus/ClassicAssist#428] [Ideas]
Has it ever been discussed or brought up the idea of having your scripts be individual files within a folder within Classic Assist instead of all thrown into a .json files within the profile?

**Comment by KevinEady** [comment]
There was a branch to introduce file system macros but I got a bit busy and couldn't finish it

https://github.com/Reetus/ClassicAssist/pull/225

**Reply by xG00BERx** [reply]
I would love to see something like this - Razor Enhanced is pretty good with this and its something different having it all in the .json

**Reply by RedSunset84** [reply]
oh yeah ... script saved to json files would be awesome for any backup or sharing ...

**Comment by RedSunset84** [comment]
Any chance we will see this feature in one if the upcomming updates @Reetus ?
